requirements.txt

```diff
@@ -8,3 +8,4 @@ rich>=13.7.1
 scipy>=1.4.1
 threadpoolctl>=3.1.0,<4.0.0
 typing-extensions>=3.7.4
+zarr>=2.5.0,<3
```

Comment:
Shouldn't be part of `requirements`, just dev ones?

Reply:
Yeah, we could make it optional without much hassle, I'd just have to do a bunch of conditional imports. The thing is that I'd like to have it included and used by default eventually.

Reply:
@ricardoV94, what do you think about this in the end? Do we make it optional?

Reply:
I would suggest making it optional until it becomes well-tested / good candidate for default?

Reply:
Did you make it optional?

Reply:
Now it's optional